src/test/java/com/google/api/codegen/testdata/discoveries/go/go_compute.v1.json.baseline

```diff
@@ -4090,13 +4090,13 @@ func main() {
     // TODO: Handle error.
   }
 
-  // Project ID for this request.
+  // * Project ID for this request.
```

Comment:
Could you add to this thread a link to the github issue where this inconsistency will be fixed?